src/redfish/discovery/discovery.py

```diff
@@ -41,7 +41,7 @@ def sanitize(number, minimum, maximum=None):
     return number
 
 
-def discover_ssdp(port=1900, ttl=2, response_time=3, iface=None, protocol="ipv4"):
+def discover_ssdp(port=1900, ttl=2, response_time=3, iface=None, address=None, protocol="ipv4"):

```

Comment:
Suggest adding the new optional param at the end
```suggestion
def discover_ssdp(port=1900, ttl=2, response_time=3, iface=None, protocol="ipv4", address=None):

```

Reply:
Fix made